bucket/dotnet-sdk.json

```diff
@@ -14,6 +14,10 @@
         }
     },
     "bin": "dotnet.exe",
+    "suggest": {
+        "Visual C++ Redistributable 2015": "extras/vcredist2015"
```

Comment:
```suggestion
        "Visual C++ Redistributable 2019": "extras/vcredist2019"
```

Reply:
The 2015 version was only included because of this: https://docs.microsoft.com/en-us/dotnet/core/install/dependencies?pivots=os-windows&tabs=netcore31#net-core-dependencies.  Not sure the 2019 version is the right choice here.

Reply:
https://support.microsoft.com/help/2977003/the-latest-supported-visual-c-downloads

Reply:
Ah. Got ya. So the vcredist2019 will cover the minimum requirements for dotnet sdk. 👍